### DISCUSSION TITLE
makeNixvim and man nixvim not appearing for default flake setup

### DISCUSSION THREAD

**Discussion by Readf0x** [nix-community/nixvim#2099] [Q&A] [answered]
Hey, I recently installed nixvim as a standalone flake using the recommended template and neither the nixvim man page or `makeNixvim` have been installed. Do I need to run it some other way than `nix run`?

**Comment by MattSturgeon** [comment, marked as answer]
`nix run` doesn't install anything. You can use `nix shell` to enter a "shell" where the package is installed, however to install things persistently, you need to add them to your system config, e.g. your nixos config or your home-manager config.

For example, in my system flake I have a standalone `packages.<system>.nvim` [flake output](https://github.com/MattSturgeon/nix-config/blob/8645b1687eda188fcc1c656ce0825315b8ef663d/nvim/flake-module.nix#L14-L17). I then pass my flake itself through to my config modules [using `specialArgs`](https://github.com/MattSturgeon/nix-config/blob/8645b1687eda188fcc1c656ce0825315b8ef663d/hosts/flake-module.nix#L3), and [install](https://github.com/MattSturgeon/nix-config/blob/8645b1687eda188fcc1c656ce0825315b8ef663d/modules/home-manager/neovim.nix#L19) `self.packages.${pkgs.system}.nvim` in my `home.packages`.

**Reply by Readf0x** [reply]
I actually initially wanted to use home-manager but the `init.lua` it generated failed to import my plugins, so I assumed it was bugged, in reality I forgot to update my flake... :facepalm:

I also misread the documentation and assumed makeNixvim was a command LOL

Anyways, thanks for the help, your config was a big help, if only because it showed me how wrong my preconceptions were. I didn't really understand flakes at the time of that post (pretty obvious, I'm sure).
